addons/mail/models/mail_activity.py

```diff
@@ -93,7 +93,6 @@ def _default_activity_type_for_model(self, model):
         default=lambda self: self.env.user,
         index=True, required=True, ondelete='cascade')
     user_tz = fields.Selection(string='Timezone', related="user_id.tz", store=True)
-    request_partner_id = fields.Many2one('res.partner', string='Requesting Partner')
```

Comment:
Note: this field is only used by website_slides, therefore it's has been moved there.

Reply:
Makes sense ^^ . But then make a specific commit for that change, it should not be hidden in your internal stuff.

Reply:
Even better, I think this change deserve it's own PR https://github.com/odoo/odoo/pull/190155

Reply:
As you prefer :)